marlowe-playground-server/marlowe-playground-server.cabal

```diff
@@ -107,31 +121,28 @@ executable marlowe-playground-server
         containers -any,
         directory -any,
         filepath -any,
-        http-client -any,
-        http-types -any,
-        playground-common -any,
         lens -any,
         marlowe -any,
         marlowe-playground-server -any,
         marlowe-symbolic -any,
-        marlowe-actus -any,
         monad-logger -any,
         mtl -any,
-        plutus-contract -any,
-        plutus-ledger,
-        plutus-tx,
+        plutus-ledger-api,
         purescript-bridge -any,
         optparse-applicative -any,
-        servant-client -any,
-        servant-foreign -any,
         servant-server -any,
         servant-purescript -any,
         text -any,
-        wai -any,
         warp -any,
-        web-ghc -any,
-        time -any,
         time-units -any
+        -- marlowe-actus -any,
+        -- plutus-tx,
+        -- servant-foreign -any,
+        -- servant-client -any,
+        -- http-client -any,
+        -- http-types -any,
+        -- wai -any,
+        -- time -any,
```

Comment:
Why leave these here?

Reply:
My bad, I commented all of them and started uncommenting the necesary ones, but forgot to delete the old ones.